src/plugins/visualizations/public/embeddable/disabled_lab_visualization.tsx

```diff
@@ -17,29 +17,39 @@
  * under the License.
  */
 
-import { FormattedMessage } from '@kbn/i18n/react';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n/react';
+import { EuiEmptyPrompt, EuiLink } from '@elastic/eui';
 import React from 'react';
+import { getDocLinks } from '../services';
 
 export function DisabledLabVisualization({ title }: { title: string }) {
+  const advancedSettingsLink = getDocLinks().links.management.visualizationSettings;
   return (
-    <div className="visDisabledLabVisualization">
-      <div
-        className="kuiVerticalRhythm visDisabledLabVisualization__icon kuiIcon fa-flask"
-        aria-hidden="true"
+    <I18nProvider>
+      <EuiEmptyPrompt
```

Comment:
Nit: can we add a `titleSize="xs"` prop to `EuiEmptyPrompt`?

Reply:
I addressed them both! Thank you 🙂 
<img width="809" alt="Screenshot 2020-11-09 at 5 47 44 PM" src="https://user-images.githubusercontent.com/17003240/98563407-d00dfc80-22b3-11eb-913a-a7c14130e762.png">